python-libraries/data-platform-catalogue/data_platform_catalogue/client/datahub/datahub_client.py

```diff
@@ -332,3 +332,6 @@ def search(
         return self.search_client.search(
             query=query, count=count, page=page, result_types=result_types
         )
+
+    def list_data_products(self) -> SearchResponse:
```

Comment:
Can you move this to the abstract base class? Since it's calling the search method it would work as a generic implementation